examples/python/bulk_get_papers_by_pmid/test

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env python
+import sys
+
+import get_papers
+
+# For occasionally verifying this script, error if we don't find enough records.
+if get_papers.count < 580:
```

Comment:
Just a little curious about this number, why 580? Are we just thinking 7 is the magic number?

Reply:
arbitrary. just don't want to be bothered for small variations in an example test script